hieradata/development.yaml

```diff
@@ -9,6 +9,8 @@ base::supported_kernel::enabled: true
 clamav::use_service: false
 
 govuk::deploy::config::asset_root: 'http://assets-origin.dev.gov.uk'
+govuk::deploy::config::csp_report_only: false
+govuk::deploy::config::csp_report_uri:
```

Comment:
Do we need this if it's defaulted?

Reply:
This is used to replace the default. I'll make it clearer with an explicit null.